client/src/components/form/EducationInfo.tsx

```
@@ -37,6 +37,19 @@ function AcademicDetails({
     field: keyof AcademicRecord,
   ) => {
     const { value } = e.target;
+    if (field === 'gpa') {
+      const gpaValue = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)